publication-commons/src/main/java/no/unit/nva/publication/model/storage/Dao.java

```diff
@@ -104,7 +105,7 @@ public void setVersion(UUID version) {
     }
 
     @Override
-    public final String getPrimaryKeyPartitionKey() {
+    public String getPrimaryKeyPartitionKey() {
```

Comment:
Removing final to be able to override this method in FileDao class, otherwise it will get the same key structure as other entities in databse, i.e. including customer and owner in primary key.